content/en/getting_started/continuous_testing/_index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Getting Started with Continuous Testing
+kind: documentation
+further_reading:
+- link: 'https://learn.datadoghq.com/courses/synthetic-tests-ci-cd-pipeline'
+  tag: 'Learning Center'
+  text: 'Introduction to Synthetic Tests in a CI/CD Pipeline'
+- link: '/synthetics/api_tests'
+  tag: 'Documentation'
+  text: 'Learn more about API tests'
+- link: '/synthetics/multistep'
+  tag: 'Documentation'
+  text: 'Learn more about multistep API tests'
+- link: '/synthetics/browser_tests'
+  tag: 'Documentation'
+  text: 'Learn more about browser tests'
+- link: '/continuous_testing/cicd_integrations'
+  tag: 'Documentation'
+  text: 'Learn about running Synthetic tests in a CI pipeline'
+algolia:
+  tags: ["continuous testing"]
+---
+{{< jqmath-vanilla >}}
+
+## Overview
+
+Continuous Testing lets you automatically run and monitor the same [Synthetic tests](https://docs.datadoghq.com/synthetics/) you’ve configured in your staging, QA, and pre-production environments to proactively alert your team and block your pipeline deployments should code changes cause testing failures. 
+
+Your codeless tests can:
+* [Launch API requests on your systems](https://docs.datadoghq.com/getting_started/synthetics/api_test)
+* [Simulate browser scenarios within your web application](https://docs.datadoghq.com/getting_started/synthetics/browser_test/)
+* [Test functionality within your iOS and Android applications](https://docs.datadoghq.com/mobile_app_testing/)
+
+Once your tests run, examine test results and CI batches in the [Synthetic Monitoring & Continuous Testing Explorer.](https://app.datadoghq.com/synthetics/explorer?track=synbatch)
+
+Improve your developer workflow with Continuous Testing by [leveraging the datadog-ci NPM Package](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm) to run these tests directly in your CI pipeline and the [Datadog Synthetics VS Code Integration](https://docs.datadoghq.com/developers/ide_integrations/) to run tests in your IDE.
+
+Datadog also offers Parallelization, allowing you to execute multiple tests in your CI/CD pipelines simultaneously rather than sequentially to help speed up your building, testing, and deployment processes.
```

Comment:
```suggestion
Continuous Testing also offers Parallelization, which allows you to execute multiple tests in your CI/CD pipelines simultaneously rather than sequentially to help speed up your building, testing, and deployment processes.
```